frb_codegen/Cargo.toml

```diff
@@ -56,3 +56,4 @@ x86_64-pc-windows-msvc = {pkg-fmt = "zip"}
 chrono = []
 default = ["chrono"]
 uuid = []
+dart-style-enums = []
```

Comment:
Well, I mean one (command line) flag, e.g. in `Opts`/`RawOpts` etc

Reply:
Yeah I removed it. Side product of just waking up :D